.drone.yml

```diff
@@ -1312,6 +1312,9 @@ steps:
     pull: always
     settings:
       auto_tag: false
+      tags:
+        - ${DRONE_TAG##v}-rootless
```

Comment:
Since this is release for release branch. this is not right I think.

Reply:
```yml
trigger:
  ref:
  - "refs/tags/**"
```

?